core/cachestub/tx_cache_stub.go

```diff
@@ -0,0 +1,100 @@
+package cachestub
+
+import (
+	"sort"
+
+	"github.com/anoideaopen/foundation/core/types"
+	"github.com/anoideaopen/foundation/core/types/big"
+	"github.com/anoideaopen/foundation/proto"
+)
+
+type TxCacheStub struct {
+	*BatchCacheStub
+	txID       string
+	txCache    map[string]*proto.WriteElement
```

Comment:
Please rename to txWriteCache

Reply:
Of course, done

Reply:
You renamed the wrong thing.
The stub should be left as it was, but the field inside the structure should be renamed